### PR TITLE
Addition of 2.2

### PR DESCRIPTION
**Pull request by dmundra**
Initial PR to include 2.2

### CODE REVIEW

**Comment by dmundra** [comment]
@maryjom and @mgifford I updated WCAG 2.2 VPAT to say 2.5 based on comment from https://github.com/CivicActions/openacr/pull/21#discussion_r1288839210.

**Comment by dmundra** [comment]
@mgifford check out the commit https://github.com/CivicActions/openacr/pull/22/commits/ba2af5fbadfab1198c6fa815acc70aeba0ceca5e to see the small adjustments I made.

**Comment by maryjom** [comment]
@mgifford @dmundra How did you intend to cover the change in WCAG 2.2 that makes 4.1.1 Parsing obsolete and deleted? This is something the VPAT committee has not yet determined how to handle, as 508 and current EN 301 549 both include it. WCAG 2.0 and 2.1 will have an errata (published soon) that this SC is always met.

**Comment by dmundra** [comment]
> @mgifford @dmundra How did you intend to cover the change in WCAG 2.2 that makes 4.1.1 Parsing obsolete and deleted? This is something the VPAT committee has not yet determined how to handle, as 508 and current EN 301 549 both include it. WCAG 2.0 and 2.1 will have an errata (published soon) that this SC is always met.

@maryjom that is a good question. Right now (just tested it) if we remove 4.1.1 from a catalog then trying to validate an openacr report will through an error. I think the catalog schema will need to support like a deprecated flag or something similar that the validator then can use to ignore that criteria and the HTML generator would ignore that output. Quick ideas though that might not be what is needed here.

File: catalog/data/wcag-2.2.yaml [hunk 1]
```diff
@@ -0,0 +1,725 @@
...
+          - authoring-tool
+      - id: 3.3.7
+        handle: Redundant Entry
+        alt_id: redundant-entry
+        components:
+          - web
+          - electronic-docs
+          - software
+          - authoring-tool
+      - id: 4.1.1
```
Comment (mgifford) [inline catalog/data/wcag-2.2.yaml:260]: So do we just delete 4.1.1 here @dmundra?

Comment (dmundra) [reply catalog/data/wcag-2.2.yaml:260]: That would make sense.

**Comment by maggiewachs** [comment]
@dmundra @mgifford any thoughts on when this may be merged? I'm updating the CMS VPAT reports and would like to use latest (v2.5 including WCAG 2.2). Merci beaucoup!

File: catalog/data/wcag-2.2.yaml (hunk 1 above)
Comment (dmundra) [reply catalog/data/wcag-2.2.yaml:260]: Not sure why I said that, previous comment (https://github.com/CivicActions/openacr/pull/22#issuecomment-1681299853) I pointed out the issue with deleting. We need to have a way to deprecate sections in the schema I think.

**Comment by dmundra** [comment]
I am going to close this in favor of getting https://github.com/GSA/openacr/pull/353 merged to support WCAG 2.2 an VPAT 2.5 version

**Pull request closed without merging**